src/glass/src/app/shared/services/auth-storage.service.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Project Aquarium's frontend (glass)
+ * Copyright (C) 2021 SUSE, LLC.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+import { Injectable } from '@angular/core';
+import * as _ from 'lodash';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthStorageService {
+  constructor() {}
+
+  set(username: string): void {
+    localStorage.setItem('username', username);
```

Comment:
where is `localStorage` coming from?

Reply:
It's global available. See https://developer.mozilla.org/de/docs/Web/API/Window/localStorage

Reply:
You can also use `window.localStorage`.